### PR TITLE
Add gjslint flags to configuration - Fixes #17

### LINKED ISSUE #17: Add config option to allow extra flags to be passed to gjslint executable
gjslint has a useful --strict flag that I'd like to be able to configure linter-gjslint to use.

We could either add support for each of the flags as separate options (nicer UI) but then that becomes a maintenance burden. So a simple text field that lets you specify extra flags that will be passed to the gjslint executable would probably be a good solution.

### PR DESCRIPTION
**Pull request by trevershick**
Add gjslint flags to configuration
Add .gitignore file
Bump version number for release.
Fixes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,11 @@ You can configure linter-gjslint by editing ~/.atom/config.cson (choose Open You
 'linter-gjslint':
   'executablePath': '' #gjslint path. run 'which gjslint' to find the path
   'gjslintIgnoreList': [] # Currently not implemented: gjslint ignore codes from http://goo.gl/OhYHYl
+  'flags': [
+    '--flagfile=.gjslintrc'
```
Comment (Arcanemagus) [inline README.md]: Missing a `,` on the end of this line.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "linter-gjslint",
   "main": "./lib/init",
-  "version": "1.0.1",
+  "version": "1.1.0",
```
Comment (Arcanemagus) [inline package.json]: Don't update the version manually, `apm publish` will do this when the update is pushed up to Atom's package manager.

Comment (trevershick) [inline package.json]: oops. ok. i've never used apm publish. i'll rebase that out. that's why i made it a separate commit.

Comment (Arcanemagus) [inline package.json]: I made the same mistake when I first started working on Atom packages, don't feel bad :wink:.

**Comment by trevershick** [comment]
OK. version bump commit has been rebased out. comma added

**Comment by Arcanemagus** [comment]
Btw if you put "Fixes #17" (or Closes or...) in the description for this it will automatically link this PR to that issue, and close that issue when this gets merged by @artoale.

**Comment by trevershick** [comment]
@Arcanemagus I put "This resolves AtomLinter/linter-gjslint#17" in the body of the first commit message message. that should do it shouldn't it?  Or does it need to be in the PR description?

[commit message](https://github.com/trevershick/linter-gjslint/commit/4c7c77c071c9aafb4c97b0129935e280bffc603c)

**Comment by Arcanemagus** [comment]
I'm pretty sure it needs to be in the PR, but I actually am not sure to be honest.

**Comment by trevershick** [comment]
i'll look into that for the next time.  sorry.

**Comment by artoale** [comment]
@trevershick thanks for your PR - looking good to me, merging!

**Pull request merged**